docs/pages/management/operations/backup-restore.mdx

```diff
@@ -137,6 +137,15 @@ $ sudo teleport start --config fresh.yaml --bootstrap state.yaml
 $ tctl --config fresh.yaml get all
 # <your state here>
 ```
+<Notice>
+The `tctl get all` is not fully supported remotely as retrieving user
```

Comment:
I don't know if I would expect anyone not familiar with Teleport to know what "not fully supported remotely" means..

Reply:
Please see update @zmb3